Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -401,20 +401,6 @@
   components:
   - type: LimitedCharges
     charges: 0
```

Comment:
Max charges needs to be restated to 30 since the component is being redeclared

Reply:
It doesn't, actually. Top-level component fields are inherited.